src/main/java/com/spotinst/sdkjava/model/bl/ocean/aks/ImportClusterDownSpecification.java

```diff
@@ -0,0 +1,63 @@
+package com.spotinst.sdkjava.model.bl.ocean.aks;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ImportClusterDownSpecification {
+
+    @JsonIgnore
+    private Set<String> isSet;
+    private Integer     maxScaleDownPercentage;
```

Comment:
This should be double. The value can be in percentages. Same for API class.

Reply:
updated in both the classes